airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -144,12 +144,15 @@ def create_stream_job(self, query: str, url: str) -> Optional[str]:
         except exceptions.HTTPError as error:
             if error.response.status_code in [codes.FORBIDDEN, codes.BAD_REQUEST]:
                 error_data = error.response.json()[0]
-                if error_data.get("message", "") == "Selecting compound data not supported in Bulk Query":
+                if (error_data.get("message", "") == "Selecting compound data not supported in Bulk Query") or (
+                    error_data.get("errorCode", "") == "INVALIDENTITY"
+                    and "is not supported by the Bulk API" in error_data.get("message", "")
+                ):
                     self.logger.error(
                         f"Cannot receive data for stream '{self.name}' using BULK API, error message: '{error_data.get('message')}'"
                     )
                 elif error.response.status_code == codes.FORBIDDEN and not error_data.get("errorCode", "") == "REQUEST_LIMIT_EXCEEDED":
-                    self.logger.error(f"Cannot receive data for stream '{self.name}', error message: '{error_data.get('message')}'")
+                    self.logger.error(f"Cannot receive data for stream '{self.name}', request limit reached: '{error_data.get('message')}'")
```

Comment:
It looks like the error message does not match the previous condition. We display that the request limit was reached but the condition check that the error data does not contain REQUEST_LIMIT_EXCEEDED.

Reply:
Thanks, will fix it